client/less/main.less

```diff
@@ -1179,4 +1179,8 @@ and (max-width : 400px)  {
 @import "jobs.less";
 @import "challenge.less";
 @import "toastr.less";
-@import "map.less";
\ No newline at end of file
+@import "map.less";
+
+ol, li {
```

Comment:
@raisesadead yes - this actually is the usual desired behavior. Somehow our ol elements haven't had bullets, which isn't desired. This remedies that.